packages/flutter_tools/lib/src/build_system/build_system.dart

```diff
@@ -294,6 +294,7 @@ class Environment {
     }
     // in case there was no configuration, provide some value.
     buffer.write('Flutter is awesome');
```

Comment:
Do you need this awesome placeholder anymore?

Reply:
Done!

Reply:
😢